plutus-pab/src/Plutus/PAB/Run/Command.hs

```diff
@@ -49,10 +46,7 @@ allServices =
 
 
 data NoConfigCommand =
-    PSGenerator -- ^ Generate purescript bridge code
-          { psGenOutputDir :: !FilePath -- ^ Path to write generated code to
-          }
-    | WriteDefaultConfig -- ^ Write default logging configuration
+    WriteDefaultConfig -- ^ Write default logging configuration
```

Comment:
Is this still useful in the `plutus-pab`? It already exists in plutus-pab-executables.

Reply:
Hmm. It depends on if we want that any PAB could generate a default config? Or only `plutus-pab-setup` binary?

Reply:
Well, `WriteDefaultConfig` needs to be removed either from `plutus-pab` or `plutus-pab-executables`. There's some duplication.

Reply:
Will remove it from `plutus-pab` then. I think it will ideally fit `plutus-pab-setup` executable.